networkit/__init__.py

```diff
@@ -161,7 +161,6 @@ def setup():
 	""" This function is run once on module import to configure initial settings """
 	setLogLevel("ERROR")  # set default loglevel for C++ code
 	setPrintLocation(True)
-	enableNestedParallelism()  # enable nested parallelism
```

Comment:
This line should not be removed, just deprecate the method. We should remove it in future releases.